### PR TITLE
Changelog 2.4 beta 2

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## 2.4.0 beta 2
+
+* Fix version constraints for Doctrine MongoDB ODM
+* Respect `_api_respond` request attribute in the SerializeListener
+* Change the normalizer's priorities (`< 0`). If you have custom normalizer to use positive priorities.
```
Comment (alanpoulain) [inline CHANGELOG.md]: Missing part?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+
+|------------------------------------------------------------|------|---------------------------------------------------------|
+| Service name                                               | Priortity | Class                                              |
```
Comment (alanpoulain) [inline CHANGELOG.md]:
```suggestion
| Service name                                               | Priority | Class                                              |
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+|------------------------------------------------------------|------|---------------------------------------------------------|
+
+* Allow custom stylesheets to be appended or replaced in the swagger UI
+* Load messenger only if available
+* Fix missing metadata cache pool for Elasticsearch
+* Make use of the new AdvancedNameConverterInterface interface for name converters
+* Refactor input/output attributes, where these attributes now take:
+	- an array specifying a class and some specific attributes (`name` and `iri` if needed)
+	- a string representing the class
+  - a `falsy` boolean to disable the input/output
```
Comment (alanpoulain) [inline CHANGELOG.md]: Indentation

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## 2.4.0 beta 2
+
+* Fix version constraints for Doctrine MongoDB ODM
+* Respect `_api_respond` request attribute in the SerializeListener
+* Change the normalizer's priorities (`< 0`). If you have custom normalizer to use positive priorities.
+
+|------------------------------------------------------------|------|---------------------------------------------------------|
```
Comment (meyerbaptiste) [inline CHANGELOG.md]: I think you failed your markdown table...

**Comment by bendavies** [comment]
Needs to be updated with my pr?

**Comment by dunglas** [comment]
Thanks @soyuka

**Pull request merged**